llvm/tools/sycl-post-link/SYCLDeviceRequirements.cpp

```diff
@@ -38,13 +36,13 @@ void llvm::getSYCLDeviceRequirements(
       {"reqd_work_group_size", "reqd_work_group_size"}};
```

Comment:
This line should probably be removed, or otherwise we are calculating `reqd_work_group_size` requirements twice